spec/model_spec.js

```diff
@@ -2143,9 +2143,10 @@ describe('Model', function () {
       });
 
       it('does not undo changes to owned associations specified in the exception option as an array', function() {
```

Comment:
@burrows While working on the only option, noticed a bug in the except option. If you write the test this way it would will fail. This is because the check was not happing in ```for (let prop in this.ownChanges) {```